src/coreclr/src/vm/excep.cpp

```diff
@@ -7473,7 +7473,7 @@ VEH_ACTION WINAPI CLRVectoredExceptionHandlerPhase3(PEXCEPTION_POINTERS pExcepti
             if ((!fAVisOk) && !(pExceptionRecord->ExceptionFlags & EXCEPTION_UNWINDING))
             {
                 PCODE ip = (PCODE)GetIP(pContext);
-                if (IsIPInModule(g_hThisInst, ip) || IsIPInModule(GCHeapUtilities::GetGCModule(), ip))
+                if (IsIPInModule(GetClrModuleBase(), ip) || IsIPInModule(GCHeapUtilities::GetGCModule(), ip))
```

Comment:
`GetGCModule` does not return module base on Unix.

Since this IsIPInModule check is Windows-only (at least today) anyway, should we make Windows-only, and ifdef the callsites for Windows?

Reply:
`GetGCModule` returns the module base for normal GC.  (it is set in `InitializeDefaultGC()`). 

Standalone variant returns hMod. Is standalone GC supported on Unix?

Reply:
I think it may be possible to pass module base for the standalone GC module. 
Just wonder if standalone GC on Unix is a real scenario. Maybe should just pass NULL on Unix/standalone case.

Reply:
There is nothing that prevents standalone GC from working on Unix.

The non-scenario on Unix is the special handling of crashes in selected modules that this code is for. As far as I know, it works just fine without any special handling. There is no SEH nor exception interop on Unix.

Reply:
I made standalone GC to pass the module base as well. I think that minimizes platform differences. 

If one day we want special case crashes in CLR/GC for reasons that include Unix as well, it will have a good chances to work on all platforms.